mteb/tasks/Classification/tur/TurkishMovieSentimentClassification.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class TurkishMovieSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TurkishMovieSentimentClassification",
+        description="Turkish Movie Review Dataset",
+        reference="https://www.win.tue.nl/~mpechen/publications/pubs/MT_WISDOM2013.pdf",
+        dataset={
+            "path": "asparius/Turkish-Movie-Review",
+            "revision": "409a4415cce5f6bcfca6d5f3ca3c408211ca00b3",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["tur-Latn"],
+        main_score="accuracy",
+        date=None,
```

Comment:
Could put a rough estimate of the dates. e.g. the upper bound could be the paper publication date.